input/pagecontent/reference.md

```diff
@@ -1294,7 +1294,7 @@ A Quantity can also be bound to a value set:
 * Set the units of the same valueQuantity to millimeters, without setting the value (assuming UCUM has been defined as an alias for http://unitsofmeasure.org):
```

Comment:
"assuming UCUM" should be "assuming $UCUM"

Reply:
Yes, fixed.